docs/react-v9/contributing/patterns/basic-recomposition.md

```diff
@@ -0,0 +1,117 @@
+## Basic Recomposition
+
+Fluent UI React V9 leans into Recomposition as one of its primary methods of customization. That is, reassemble parts of components to inherit most behaviors but also change some. We wanted to change 2 things for the Nav Divider component to help consumers quickly and easily implement specified behaviors:
```

Comment:
I would say that the `Nav` is an example of what is being talked about above.

```suggestion
Fluent UI React V9 leans into recomposition as one of its primary methods of customization. That is, reassemble parts of components to inherit most behaviors but also change some. Below, we will go through an example where we needed to do this in the library with the `NavDivider` component.

We wanted to change 2 things for the `NavDivider` component, when compare to the regular `Divider`, to help consumers quickly and easily implement specified behaviors:
```

Reply:
Changed the sentence structure slightly to prefer an active voice and shortened 🙂